issues_to_socrata.py

```diff
@@ -32,12 +33,22 @@ def extract_workgroups_from_labels(labels):
     return ", ".join(workgroup_labels_no_prefix) or None
 
 
-def get_github_issues(repo_name, github_access_token, state="all"):
+def get_github_issues(repo_name, github_access_token, state="open"):
```

Comment:
@RR216 This changes this script to only retrieve Github issues that have a status of "open" instead of all of them. [Here is the API reference for that parameter](https://docs.github.com/en/free-pro-team@latest/rest/issues/issues?apiVersion=2022-11-28#list-repository-issues). I think we still intend to fill the open dataset with GitHub issues that have either open or closed status so could you update this back to all? Thanks! 🙏

Reply:
Thanks for flagging this Mike! Yesterday the script was going slow on Rachel's machine so I told her to test on "open" issues to speed up development. You are right, it needs to be updated back to "all".

Reply:
@mddilley Yup - change applied

Reply:
oh yep - @RR216 please push that change before merge 🙏 Thanks for catching that, Rose! 🔍

Reply:
@roseeichelmann @mddilley so sorry! I retracted some code due to a bug and forgot to update the state. now done!